.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+node_js:
+  - "8"
+  - "10"
+  - "node"
+sudo: false
+install:
+  - npm install
+script:
+  - if [ "x$BUNDLER" = "x" ]; then npm run test; fi
+  - if [ "x$BUNDLER" != "x" ]; then npm run test-karma; fi
+# only run karma tests for one node version
+matrix:
+  include:
+  - name: "Browser Unit Tests (webpack)"
+    node_js: "10"
+    env: BUNDLER=webpack
+  - name: "Browser Unit Tests (browserify)"
```

Comment:
Either comment out browserify support here or add browserify and karma-browserify devdeps.